src/content/reference/faqs.mdx

```diff
@@ -246,3 +246,13 @@ You can only create `Roles` restricted to the endpoints above. `RoleBindings` ca
 This is related to how syncthing interacts with `tsc`. Syncthing creates a temporary file and replaces the original file with the new one.
 
 To solve the problem you just add the flag `--synchronousWatchDirectory` to your `tsc` command.
+
+## I cannot connect to the Kubernetes cluster using the kubeconfig file generated by Okteto
```

Comment:
Added this FAQ as I think it might be a common problem and I will add a post in the community as soon as the changes are available.

Reply:
Is this needed for any command or just for "okteto kubeconfig" @ifbyol ?

Reply:
One clarification, this is not needed to run `okteto kubeconfig`. It is needed to run any command executed with `kubectl` or similar tool using a kubeconfig generated by Okteto via `okteto kubeconfig` or downloaded from the UI. Internal connections to the cluster by the CLI will be managed in a different way. It doesn't require to have the CLI in the path.

Reply:
Can we mention that on the Kubeconfig command doc? And also in the doc that talks about downloading your Kubeconfig from the UI?

Reply:
The other file modified in the PR is the one that talks about the 2 options to get the Kubernetes config from Okteto. In the `Using your Kubernetes credentials` section, we are adding that. How would you make it more visible? A special call out with special markdown? We will include it also in the command docs

Reply:
I changed it to mention it also in the `kubeconfig` command docs and the getting started. I also sent a PR in the CLI to mention it in the command help